apidef/oas/authentication.go

```diff
@@ -482,7 +482,7 @@ type Scopes struct {
 	// Tyk classic API definition:
 	// - For OIDC: `scopes.oidc.scope_claim_name`
 	// - For JWT: `scopes.jwt.scope_claim_name`
-	ClaimName string `bson:"claimName,omitempty" json:"claimName,omitempty"`
+	ClaimName []string `bson:"claimName,omitempty" json:"claimName,omitempty"`
```

Comment:
Hey @kofoworola, wouldn't this change be considered a breaking change?

Reply:
this is valid, i will bring it up for discussion

Reply:
Alright, maybe we can have a custom UnmarshalJSON implementation that handles it, but for that ClaimName should be interface{}.